napari/layers/base/base.py

```diff
@@ -720,11 +730,7 @@ def selected(self, selected):
         if selected == self.selected:
             return
         self._selected = selected
-
-        if selected:
-            self.events.select()
-        else:
-            self.events.deselect()
+        self.events.selection(value=selected)
```

Comment:
I know all this selected stuff may go away, but in so much as it is still here if the attribute is named `selected`, we probably want the event to be `selected` instead of `selection` for consistency with the Pydantic eventedmodel approach. maybe we can get all this sorted out ahead of time in it's own PR though, that builds off of #1952 too

Reply:
oh yeah, thanks will fix